sources/packages/web/src/services/http/CASSupplier.ts

```diff
@@ -21,6 +21,17 @@ export class CASSupplierApi extends HttpBaseClient {
     );
   }
 
+  /**
+   * Retries CAS supplier info for a student.
+   * @param studentId student id.
+   */
+  async retryCASSupplier(studentId: number): Promise<void> {
```

Comment:
This will return a `PrimaryIdentifierAPIOutDTO` from the API, please adjust the return type.